src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +183,18 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface FinishBuildTrigger {
+    val initiatedBuildConfiguration: BuildConfigurationId
+    val afterSuccessfulBuildOnly: Boolean
+    val branchFilter: List<BranchRule>
```

Comment:
It may look strange that a property named `branchFilter` contains a list.
BTW how this property is used? Maybe it would be simpler to replace it by separate `excludedBranchePatterns` and `includedBranchPatterns` properties of type `Set<String>`, this way we'll get rid of an additional interface and simplify code. What do you think?